bigtable-dataflow-parent/bigtable-beam-import/src/main/java/com/google/cloud/bigtable/beam/sequencefiles/CreateTableHelper.java

```diff
@@ -0,0 +1,92 @@
+package com.google.cloud.bigtable.beam.sequencefiles;
```

Comment:
Header, please (And use Google LLC instead of Google Inc.)

Reply:
done

Reply:
Please use Google LLC instead of Google Inc, as per mbrukman.